xsnippet_api/resources/snippets.py

```diff
@@ -56,5 +251,16 @@ class Snippets(resource.Resource):
 
     async def post(self):
         snippet = await self.read_request()
+
+        v = cerberus.Validator(_SCHEMA)
```

Comment:
I'm not sure about it. Take a look on how I retrieve errors below: I use `.error` attribute of validator, hence - it's mutable. I'm not sure it's cleared on each validation call, so I decided to create it from scratch each time.

Still, I need to investigate it.